examples/timer.effekt

```diff
@@ -0,0 +1,5 @@
+def main() = {
+    var time = currentTimeNanos();
+    var timeMillis = time / 1000000;
+    println("time millis: " ++ show(timeMillis))
+}
```

Comment:
printing the time in a "test" is difficult because you don't know what the expected result will be